source/NVDAObjects/UIA/sysListView32.py

```diff
@@ -0,0 +1,105 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2023 NV Access Limited, Leonard de Ruijter
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+
+"""Module for native UIA implementations of SysListView32, e.g. in Windows Forms."""
+
+from comtypes import COMError
+import config
+from config.configFlags import ReportTableHeaders
+import UIAHandler
+from ..behaviors import RowWithFakeNavigation
+from . import ListItem, UIA
+
+
+def findExtraOverlayClasses(obj, clsList):
+	UIAControlType = obj.UIAElement.cachedControlType
+	if UIAControlType == UIAHandler.UIA.UIA_ListItemControlTypeId:
+		clsList.insert(0, SysListViewItem)
+	if UIAControlType == UIAHandler.UIA.UIA_ListControlTypeId:
+		clsList.insert(0, SysListViewList)
+
+
+class SysListViewList(UIA):
+	...
+
+
+class SysListViewItem(RowWithFakeNavigation, ListItem):
+
+	def _get_name(self):

```

Comment:
please add the return type

Reply:
Shouldn't this be inferred from the base?